engine/src/main/java/org/terasology/entitySystem/event/internal/EventSystemImpl.java

```diff
@@ -263,12 +263,10 @@ public void send(EntityRef entity, Event event) {
     }
 
     private void sendStandardEvent(EntityRef entity, Event event, List<EventHandlerInfo> selectedHandlers) {
-        for (EventHandlerInfo handler : selectedHandlers) {
```

Comment:
This method is called very often. I would leave its code as-is, because I think that the simple iteration is more performant than the conversion and lambda-based filtering.